tests/fixtures/node-module-peer-optional-none/function.js

```diff
@@ -1,6 +1,6 @@
-try {
-  // eslint-disable-next-line import/no-unassigned-import, import/no-unresolved, node/global-require
-  require('consistent-ids')
-} catch (error) {}
+// eslint-disable-next-line import/no-unresolved
+const module = require('consistent-ids')
+
+console.log(module)
```

Comment:
Added a `console.log` so that the module isn't removed in the tree shaking phase.